Comp2025/src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -385,11 +397,11 @@ private void moveToPositionInit(double newHeight, boolean holdPosition)
     m_mmMoveTimer.restart( );
     m_mmHardStopCounter = 0;
 
-    if (!(m_leftCalibrated && m_rightCalibrated))
-    {
-      DataLogManager.log(String.format("%s: MM Position move target %.1f in - NOT CALIBRATED!", getSubsystem( ), m_targetHeight));
-      return;
-    }
+    // if (!(m_leftCalibrated && m_rightCalibrated))
```

Comment:
Since this would only run if we scheduled the calibration command, there is no reason to comment out these lines, right?